gridcomps/History3G/HistoryCollectionGridComp_private.F90

```diff
@@ -31,4 +33,31 @@ function make_geom(hconfig, rc) result(geom)
       _RETURN(_SUCCESS)
    end function make_geom
 
+   !subroutine make_import_state(gridcomp, hconfig, rc)
```

Comment:
This procedure will need a better name.  (I know it is commented out for now.)   It is not making the ESMF Import State per se.  Rather it is registering import items.  So maybe `register_imports()`?

Reply:
@tclune, ok , nothing has been implemented yet so we can call it whatever we want.